api/src/Modules/staff/ModuleStaff.php

```diff
@@ -40,5 +42,76 @@ public function handle(Request $request, Response $response, $data, $code)
     }
 
 
+    public function customizeAnnouncementRBAC($instance, $database)
+    {
+        $role = $instance->getRole('all.staff');
+        $role->addPermission('api.get.staff');
+
+        if (array_key_exists('IS_ADMIN', $_SESSION) && $_SESSION['IS_ADMIN']) {
+            $adminRole = $instance->getRole('admin');
+            $adminRole->addPermission('api.post.staff.all');
+            $adminRole->addPermission('api.put.staff.all');
+            $adminRole->addPermission('apu.delete.staff.all');
```

Comment:
All of this is the 'old' way...  What you want is to be doing 

`$instance->grantPermission($role, $perm);`